src/lib/config.ts

```diff
@@ -59,6 +59,14 @@ export function intVar<R>(varName: string, defaultValue?: R): number | R {
 }
 
 export const API_HOST = requiredVar('API_HOST');
+export const API_HEARTBEAT_STATE_ACTIVE = intVar(
+	'API_HEARTBEAT_STATE_ACTIVE',
```

Comment:
Can you rename this
```suggestion
	'API_HEARTBEAT_STATE_ENABLED',
```
or
```suggestion
	'ENABLE_API_HEARTBEAT_STATE',
```